test_mock_rimesync.rb

```diff
@@ -0,0 +1,629 @@
+require 'test/unit'
+require_relative 'rimesync'
+
+# rubocop:disable ClassLength
+class TestMockRimeSync < Test::Unit::TestCase # :nodoc:
+  def setup
+    baseurl = 'http://ts.example.com/v0'
+    @ts = TimeSync.new(baseurl, token=nil, test = true)
+    @ts.authenticate('testuser', 'testpassword', 'password')
```

Comment:
You need to provide the names for the keyword args, like so: `@ts.authenticate(username: 'testuser', password: 'testpassword' ...`